packages/php-datatypes/src/Definition/Bigquery.php

```diff
@@ -92,8 +107,16 @@ class Bigquery extends Common
 
     public const MAX_LENGTH = 9223372036854775807;
 
+    /** @phpstan-var BigqueryTableFieldSchema|null */
+    private ?array $fieldAsArray = null;
+
     /**
-     * @param array{length?:string|null, nullable?:bool, default?:string|null} $options
+     * @phpstan-param array{
+     *     length?:string|null,
+     *     nullable?:bool,
+     *     default?:string|null,
+     *     fieldAsArray?:BigqueryTableFieldSchema
```

Comment:
Do datatype sem přidal fieldAsArray kdyby to bylo potřeba

Reply:
`BigqueryTableFieldSchema` a u toho je komentář `structure returned by BQ REST API, fields is recursive but this is not supported by phpstan`